app/scripts/controllers/newOverview.js

```diff
@@ -1221,15 +1225,22 @@ function OverviewController($scope,
         resource: 'instances'
       }, context, function(serviceInstances) {
         state.serviceInstances = serviceInstances.by('metadata.name');
+        updateFilter();
       }, {poll: limitWatches, pollInterval: DEFAULT_POLL_INTERVAL}));
     }
 
     if(Constants.ENABLE_TECH_PREVIEW_FEATURE.service_catalog_landing_page) {
+
+      DataService.list("secrets", context, null, { errorNotification: false }).then(function(secretData) {
```

Comment:
Hmm... so `DEFAULT_POLL_INTERVAL` is 60 seconds.  I like being consistent with our polling if we don't watch, but that seems a long time to wait if things change.  Do we want to do the logic of #2 or have a 2nd slightly more frequent poll interval?